R/backport_linter.R

```diff
@@ -45,12 +45,13 @@ backport_linter <- function(r_version = getRversion(), except = character()) {
   backport_index <- rep(names(backport_blacklist), times = lengths(backport_blacklist))
   names(backport_index) <- unlist(backport_blacklist)
 
-  names_xpath <- "//SYMBOL | //SYMBOL_FUNCTION_CALL"
-
   Linter(linter_level = "expression", function(source_expression) {
     xml <- source_expression$xml_parsed_content
 
-    all_names_nodes <- xml_find_all(xml, names_xpath)
+    all_names_nodes <- combine_nodesets(
+      source_expression$xml_find_function_calls(NULL),
```

Comment:
may as well build in the idea from #2365 here? then we can mark that PR as superseded+close when this is merged.

Reply:
done.